src/toast/ops/pointing_detector/pointing_detector.py

```diff
@@ -46,6 +47,17 @@ class PointingDetectorSimple(Operator):
         defaults.boresight_radec, help="Observation shared key for boresight"
     )
 
+    hwp_angle = Unicode(
+        defaults.hwp_angle, allow_none=True, help="Observation shared key for HWP angle"
+    )
+
+    hwp_deflection_radius = Quantity(
+        None,
+        allow_none=True,
+        help="If non-zero, nominal detector pointing will be deflected in a circular "
+        "pattern according to HWP phase.",
+    )
```

Comment:
I think we also need a `hwp_deflection_phase` here (default zero), since the deflection is not necessarily aligned with the fast axis of the HWP.

Reply:
Please see the revised code. I also added a unit test